pom.xml

```diff
@@ -1,110 +1,110 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.jenkins-ci.plugins</groupId>
-		<artifactId>plugin</artifactId>
-		<version>1.554.2</version>
-	</parent>
-	<artifactId>build-pipeline-plugin</artifactId>
-	<version>1.4.8-SNAPSHOT</version>
-	<packaging>hpi</packaging>
-	<name>Build Pipeline Plugin</name>
-	<description>This plugin provides build pipeline functionality to Hudson and Jenkins.  This allows a chain of jobs to be visualised in a new view.  Manual jobs in the pipeline can be triggered by a user with the appropriate permissions manually confirming.</description>
-	<url>https://wiki.jenkins-ci.org/display/JENKINS/Build+Pipeline+Plugin</url>
-	<licenses>
-		<license>
-			<name>MIT License</name>
-			<url>http://www.opensource.org/licenses/mit-license.php</url>
-		</license>
-	</licenses>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>1.554.3</version>
```

Comment:
@amuniz No, it was not but 1.554.3 provides fixes interesting. it was a way to promote the last LTS on 1.554